com.unity.render-pipelines.universal/Runtime/Debug/DebugDisplaySettingsMaterial.cs

```diff
@@ -134,50 +142,100 @@ struct AlbedoDebugValidationPresetData
+        /// <summary>
+        /// Current minimum luminance threshold value for albedo validation.
+        /// Any albedo luminance values below this value will be considered invalid and will appear red on screen.
+        /// </summary>
+        public float albedoMinLuminance = 0.01f;
+
+        /// <summary>
+        /// Current maximum luminance threshold value for albedo validation.
+        /// Any albedo luminance values above this value will be considered invalid and will appear blue on screen.
+        /// </summary>
+        public float albedoMaxLuminance = 0.90f;
 
         float m_AlbedoHueTolerance = 0.104f;
-        internal float AlbedoHueTolerance
+
+        /// <summary>
+        /// Current hue tolerance value for albedo validation.
+        /// </summary>
+        public float albedoHueTolerance
         {
-            get => m_AlbedoDebugValidationPreset == AlbedoDebugValidationPreset.DefaultLuminance ? 1.0f : m_AlbedoHueTolerance;
+            get => m_AlbedoValidationPreset == AlbedoDebugValidationPreset.DefaultLuminance ? 1.0f : m_AlbedoHueTolerance;
             private set => m_AlbedoHueTolerance = value;
         }
 
         float m_AlbedoSaturationTolerance = 0.214f;
-        internal float AlbedoSaturationTolerance
+
+        /// <summary>
+        /// Current saturation tolerance value for albedo validation.
+        /// </summary>
+        public float albedoSaturationTolerance
         {
-            get => m_AlbedoDebugValidationPreset == AlbedoDebugValidationPreset.DefaultLuminance ? 1.0f : m_AlbedoSaturationTolerance;
+            get => m_AlbedoValidationPreset == AlbedoDebugValidationPreset.DefaultLuminance ? 1.0f : m_AlbedoSaturationTolerance;
             private set => m_AlbedoSaturationTolerance = value;
         }
 
-        internal Color AlbedoCompareColor = new Color(127f / 255f, 127f / 255f, 127f / 255f, 255f / 255f);
+        /// <summary>
+        /// Current target color value for albedo validation.
+        /// </summary>
+        public Color albedoCompareColor = new Color(127f / 255f, 127f / 255f, 127f / 255f, 255f / 255f);
+
+        /// <summary>
+        /// Current minimum threshold value for metallic validation.
+        /// Any metallic values below this value will be considered invalid and will appear red on screen.
+        /// </summary>
+        public float metallicMinValue = 0.0f;
 
-        internal float MetallicMinValue = 0.0f;
-        internal float MetallicMaxValue = 0.9f;
+        /// <summary>
+        /// Current maximum threshold value for metallic validation.
+        /// Any metallic values above this value will be considered invalid and will appear blue on screen.
+        /// </summary>
+        public float metallicMaxValue = 0.9f;
 
-        internal DebugMaterialValidationMode MaterialValidationMode;
+        /// <summary>
+        /// Current material validation mode.
+        /// </summary>
+        public DebugMaterialValidationMode materialValidationMode;
```

Comment:
Would it make sense to have these public variables as Properties?

Reply:
Thanks, I think this is a good idea. I'll update this

Reply:
👍